concepticondata/conceptlists/Bowern-2012-348.tsv

```diff
@@ -0,0 +1,349 @@
+NUMBER	ID	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS	COMMENTS
+1	Bowern-2012-348-1	1dl.excl	2636	WE TWO (EXCLUSIVE)	
+2	Bowern-2012-348-2	1dl.incl	2637	WE TWO (INCLUSIVE)	
+3	Bowern-2012-348-3	3dl	2635	THEY TWO	
+4	Bowern-2012-348-4	Bloodwood	3007	EUCALYPT	
+5	Bowern-2012-348-5	Blue-Tongue Lizard	632	LIZARD	mapping to LIZARD as it's the only Lacertilia in the list
+6	Bowern-2012-348-6	Brolga	1830	CRANE	mapping to CRANE as it's the only Gruidae in the list
+7	Bowern-2012-348-7	Carpet snake	1563	PYTHON	mapping to PYTHON as it's the only Pythonidae in the list
+8	Bowern-2012-348-8	Crow	913	CROW	
+9	Bowern-2012-348-9	Death adder	3008	DEATH ADDER	
+10	Bowern-2012-348-10	Duck	1360	DUCK	
+11	Bowern-2012-348-11	Echidna	3009	ECHIDNA	
+12	Bowern-2012-348-12	Emu	2817	EMU	
+13	Bowern-2012-348-13	I	1209	I	
+14	Bowern-2012-348-14	Magpie	2039	MAGPIE	
+15	Bowern-2012-348-15	Nest	539	NEST	
+16	Bowern-2012-348-16	Possum	470	POSSUM	
+17	Bowern-2012-348-17	Scorpion	1538	SCORPION	
+18	Bowern-2012-348-18	Shield	1064	SHIELD	
+19	Bowern-2012-348-19	Sulphur-Crested Cockatoo	3010	WHITE COCKATOO	
+20	Bowern-2012-348-20	Willy Wagtail	3011	WAGTAIL	
+21	Bowern-2012-348-21	Witchetty Grub	3012	GRUB	
+22	Bowern-2012-348-22	above	1741	ABOVE	
+23	Bowern-2012-348-23	again	204	AGAIN	
+24	Bowern-2012-348-24	all	98	ALL	
+25	Bowern-2012-348-25	alone	1964	ALONE	
+26	Bowern-2012-348-26	and	1577	AND	
+27	Bowern-2012-348-27	angry	1826	ANGRY	check if it shouldn't be BE ANGRY
+28	Bowern-2012-348-28	ankle	803	ANKLE	
+29	Bowern-2012-348-29	ant	587	ANT	
+30	Bowern-2012-348-30	armpit	1886	ARMPIT	
+31	Bowern-2012-348-31	ash	646	ASH	
+32	Bowern-2012-348-32	ask	1309	ASK (INQUIRE)	check if it should be ASK (REQUEST)
+33	Bowern-2012-348-33	at	1461	AT	
+34	Bowern-2012-348-34	back	1291	BACK	
+35	Bowern-2012-348-35	bad/evil	45	EVIL	
+36	Bowern-2012-348-36	bag	2468	BAG	
+37	Bowern-2012-348-37	bandicoot	3013	BANDICOOT	
+38	Bowern-2012-348-38	bat	1793	BAT	
+39	Bowern-2012-348-39	beard	678	BEARD	
+40	Bowern-2012-348-40	belly	1251	BELLY	
+41	Bowern-2012-348-41	below	2094	DOWN OR BELOW	
+42	Bowern-2012-348-42	big	1202	BIG	
+43	Bowern-2012-348-43	bird	937	BIRD	
+44	Bowern-2012-348-44	bite	1403	BITE	
+45	Bowern-2012-348-45	black	163	BLACK	
+46	Bowern-2012-348-46	black cockatoo	3014	BLACK COCKATOO	
+47	Bowern-2012-348-47	black duck	3015	BLACK DUCK	
+48	Bowern-2012-348-48	blind	1653	BLIND	
+49	Bowern-2012-348-49	blood	946	BLOOD	
+50	Bowern-2012-348-50	blow	175	BLOW (OF WIND)	needs to be confirmed
```

Comment:
we often have these cases, and I remember that there was confusion in the original dataset, as they did not always distinguish between what "blow" could mean. We may consider: a) adding a meta-concetp (blow (with mouth or of wind)), but this is tedious, or we make the bold decision to assume it is b) blow (of wind). I'd be inclined to go with b), as this is a very data-specific problem.

Reply:
Good. I am collecting the doubts to ask experts later on, but I tried to find out or infer the concept for these cases by looking at the data or with some google-fu. "blow", if I remember correctly, was one of the cases I was able to find, even though with a low confidence level.

Reply:
In chirila, the database which will supersedes this dataset at some point in the future, there are dubious cases, I remember, but I also think that Claire wanted to fix them. I am fine with linking to "blow" now, as it was surely intended to mean "blow of wind", neither the blow as a beat or the blow of the mouth or the trumpet.